src/components/network/network-map.js

```diff
@@ -54,6 +55,10 @@ const NetworkMap = forwardRef((props, ref) => {
     const [tooltip, setTooltip] = useState({});
 
     const theme = useTheme();
+
+    const [substationLoaded, setSubstationLoaded] = useState(false);
```

Comment:
substationsLoaded for state name and setSubstationsLoaded for setter name

Reply:
done